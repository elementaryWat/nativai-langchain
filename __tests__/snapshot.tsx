import { render } from "@testing-library/react";
import Home from "@/pages/index";

jest.mock("next-auth/react", () => ({
  useSession: () => [
    { session: { user: { email: "test@example.com" } } },
    false,
  ],
}));
jest.mock("react-redux", () => ({
  useDispatch: () => jest.fn(),
  useSelector: () => jest.fn(),
}));

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
