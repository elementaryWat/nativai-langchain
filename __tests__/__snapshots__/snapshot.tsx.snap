// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    background="linear-gradient(180deg, #6e45ff, black)"
    class="MuiGrid-root css-wv9ayh-MuiGrid-root"
  >
    <div>
      <div
        class="signin_container"
      >
        <div
          class="hero_login"
        >
          <img
            alt="Logo"
            data-nimg="1"
            decoding="async"
            height="40"
            loading="lazy"
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style="color: transparent;"
            width="40"
          />
        </div>
        <div
          class="container_button"
        >
          <button
            class="butto_signin"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M15.545 6.558a9.42 9.42 0 0 1 .139 1.626c0 2.434-.87 4.492-2.384 5.885h.002C11.978 15.292 10.158 16 8 16A8 8 0 1 1 8 0a7.689 7.689 0 0 1 5.352 2.082l-2.284 2.284A4.347 4.347 0 0 0 8 3.166c-2.087 0-3.86 1.408-4.492 3.304a4.792 4.792 0 0 0 0 3.063h.003c.635 1.893 2.405 3.301 4.492 3.301 1.078 0 2.004-.276 2.722-.764h-.003a3.702 3.702 0 0 0 1.599-2.431H8v-3.08h7.545z"
              />
            </svg>
             
            <span>
              Inicia Sesion
            </span>
          </button>
        </div>
        <div
          class="circulos_signin"
        >
          <div
            class="circulo_signin1"
          />
          <div
            class="circulo_signin2"
          />
          <div
            class="circulo_signin3"
          />
          <div
            class="circulo_signin4"
          />
          <div
            class="circulo_signin5"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-1tktdf3-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-1v5v5qq"
      >
        <div
          class="MuiGrid-root css-1rrj52y-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-7hjqdy-MuiGrid-root"
          >
            <img
              alt="Logo"
              data-nimg="1"
              decoding="async"
              height="40"
              loading="lazy"
              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
              style="color: transparent;"
              width="40"
            />
            <h5
              class="MuiTypography-root MuiTypography-h5 css-1fy77fu-MuiTypography-root"
            >
              Â¡Empieza ahora!
            </h5>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiMobileStepper-root MuiMobileStepper-positionStatic css-55425n-MuiPaper-root-MuiMobileStepper-root"
      >
        <div
          class="MuiMobileStepper-dots css-1384nzf-MuiMobileStepper-dots"
        >
          <div
            class="MuiMobileStepper-dot MuiMobileStepper-dotActive css-26w9jf-MuiMobileStepper-dot"
          />
          <div
            class="MuiMobileStepper-dot css-114p2tk-MuiMobileStepper-dot"
          />
          <div
            class="MuiMobileStepper-dot css-114p2tk-MuiMobileStepper-dot"
          />
          <div
            class="MuiMobileStepper-dot css-114p2tk-MuiMobileStepper-dot"
          />
        </div>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall css-nkyh4g-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="KeyboardArrowRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</div>
`;
